src/plot/plaxis.cpp

```diff
@@ -669,14 +669,25 @@ void wxPLTimeAxis::RecalculateTicksAndLabel() {
     //We need to figure out whether we are looking at hours, days, or months, and label the graph appropriately.
     wxDateTime timeKeeper(1, wxDateTime::Jan, 1971, 0, 0, 0); // works all time zones
 
+
     double world_len = m_max - m_min;
     double time = m_min;
+
+    // to skip leap years (SAM issue 289) - scale to first year
+    int m_min_offset = m_min / 8760;
+    double m_min_scaled = (double)((int)m_min % 8760); // m_min and m_max always based on 8760 (365 days with no leap years)
```

Comment:
Likely a different issue, but there's hopes to actually handle leap days in the calculations this year. This would likely require an additional flag to this function to scale to 8784 if a leap year is detected, and base each year on a leap year instead (8784 * timesteps_per_hour * nyears).

Reply:
> Likely a different issue, but there's hopes to actually handle leap days in the calculations this year. This would likely require an additional flag to this function to scale to 8784 if a leap year is detected, and base each year on a leap year instead (8784 * timesteps_per_hour * nyears).

Good catch and way to use existing code. If we do leap years, we can probably assign a base year and then use the full wxDateTime class to handle as it was designed.